Kibol-Alert/ViewModels/UserVM.cs

```diff
@@ -1,15 +1,11 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
-
-namespace Kibol_Alert.ViewModels
+﻿namespace Kibol_Alert.ViewModels
 {
     public class UserVM
     {
         public int UserId { get; set; }
         public string UserName { get; set; }
         public string Email { get; set; }
+        public string FavoriteClub { get; set; }
         public bool IsDeleted { get; set; } = false;
```

Comment:
We dont need IsDeleted in ViewModel, because it will be checked at backend, where we will have still 'User'.
In ViewModel we want to have just info, which will be sent to frontend